agent/agent.go

```diff
@@ -228,21 +228,9 @@ func New(c *Config) (*Agent, error) {
 		httpAddrs:       httpAddrs,
 		tokens:          new(token.Store),
 	}
-	if err := a.resolveTmplAddrs(); err != nil {
```

Comment:
We sort of designed tagged addresses with opening up as possible, but we don't have plans to do so right now.